src/wasm.h

```diff
@@ -32,6 +32,7 @@
 #include <string>
 #include <vector>
 
+#include "asmjs/shared-constants.h"
```

Comment:
why is this needed? if the metadce cpp change needs it, include it there maybe?

Reply:
Sorry, not sure how that happened!